test/views/templates/returns/PaymentStatusTemplateSpec.scala

```diff
@@ -29,120 +29,223 @@ class PaymentStatusTemplateSpec extends TemplateBaseSpec {
 
   val whatYouOweCalc = "We have worked out what you owe based on any payments you might have made on your account. "
 
-  "Rendering the payment status information" when {
+  val whatYouOweCalcAgent = "We have worked out what your client owes based on any payments made on their account. "
+
+  "A principal user" should {
+    "render the payment status information" when {
+
+      "then User is NOT Hybrid and" when {
```

Comment:
Could you please fix this typo in the test description (should be 'the user' instead of 'then user')?